### ISSUE TITLE
An invalid parsed json when txt-matched-rules-extended in CRS sandbox detected as PL4 detected it

### ISSUE THREAD

**Issue by HackingRepo** [coreruleset/coreruleset#4787]
To reproduce that specific bug, i used burp, and send that req

```rust
POST / HTTP/1.1
Host: sandbox.coreruleset.org
User-Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:154.0) Gecko/20100101 Firefox/154.0
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Language: en-US,en;q=0.9
Accept-Encoding: gzip, deflate, br
Upgrade-Insecure-Requests: 1
Sec-Fetch-Dest: document
Sec-Fetch-Mode: navigate
Sec-Fetch-Site: none
Sec-Fetch-User: ?1
x-crs-version: latest
x-format-output: txt-matched-rules-extended
Priority: u=0, i
Te: trailers
Connection: keep-alive
Content-Length: 7
Content-Type: application/json


}

```

and then as i observed in the response it says

```rust
HTTP/1.1 400 Bad Request
Date: Tue, 08 Sep 2026 07:05:29 GMT
Content-Type: text/plain; charset=iso-8859-1
Connection: keep-alive
X-Unique-ID: ap-zucjOskoRZsf1I0c_vAAAAIg
x-backend: apache-latest
Content-Length: 325

This payload has been tested against the OWASP CRS 
web application firewall. The test was executed using the apache engine and CRS version latest.

The payload is being detected by triggering the following rules:

200002 PL? Failed to parse request body.


CRS therefore detects this payload starting with paranoia level 4.
```

it claims that the payload detected starting from PL4, that just a parsing failure, that is a bug that needs be fixed

**Comment by fzipi** [comment]
It is just cosmetics.

**Comment by fzipi** [comment]
Should be fixed.
